extension.ts

```diff
@@ -328,6 +328,22 @@ const createRecentItem = (kind: RecentItemKind) => (
   };
 };
 
+/**
+ * Convert any workspace item objects to URIs.
+ *
+ * @param item The workspace item from storage.json
+ * @returns The URI for the workspace
+ */
+const workspaceItemToUri = (
+  item: string | { configURIPath: string }
```

Comment:
Please extract this into a dedicated type, e.g.

```typescript
interface MultiRootWorkspaceItem {
  readonly configURIPath: string;
}

type WorkspaceItem = MultiRootWorkspaceItem | string;
```

Reply:
:+1: